unabridged-works-volume-3/original/15-letters.adoc

```diff
@@ -4937,13 +5212,17 @@ and to do the will of God,--he doth not yet at all know
 the Lord Jesus Christ unto the salvation of his soul,
 notwithstanding whatsoever he may profess or believe concerning him.
 
+[.signed-section-signature]
 I+++.+++ P.
 
+[.signed-section-context-close]
 Astrop, 28th of Sixth Month, 1678
 
-TO THE WOMEN FRIENDS THAT MEET AT ARMSCOT IN WORCESTERSHIRE
+[.letter-heading]
+To the Women Friends That Meet at Armscot in Worchestershire
```

Comment:
You accidentally snuck in an extra `h` in `Worcestershire`.

Reply:
Is that how the sauce is spelled?! I was thinking about the sauce when I was rewriting it. lol